scripts/bootstrap/compile.sh

```diff
@@ -41,7 +41,7 @@ if [ "$ERROR_PRONE_INDEX" -lt "$GUAVA_INDEX" ]; then
   TEMP_FOR_SWAP="${LIBRARY_JARS_ARRAY[$ERROR_PRONE_INDEX]}"
   LIBRARY_JARS_ARRAY[$ERROR_PRONE_INDEX]="${LIBRARY_JARS_ARRAY[$GUAVA_INDEX]}"
   LIBRARY_JARS_ARRAY[$GUAVA_INDEX]="$TEMP_FOR_SWAP"
-  LIBRARY_JARS="${LIBRARY_JARS_ARRAY[@]}"
+  LIBRARY_JARS="${LIBRARY_JARS_ARRAY[*]}"
```

Comment:
I wonder how this could have passed before. See [SC2124](https://github.com/koalaman/shellcheck/wiki/SC2124) for reference.

Reply:
I think this was made on purpose, I cannot remember why though.

Reply:
Seems like `LIBRARY_JARS` is only used [in this line](https://github.com/bazelbuild/bazel/blob/master/scripts/bootstrap/compile.sh#L84) and I am pretty sure this expects a single value (concatenated via spaces) rather than an array.

Reply:
Indeed.